stable/openstack-base/README.md

```diff
@@ -160,17 +135,20 @@ boot in UEFI mode:
 
 ## Configure networking
 
-Neutron networking will be configured with the aid of two scripts supplied by
+Networking will be configured with the aid of two scripts supplied by
 the bundle.
 
 > **Note**: These scripts are written for Python 2 and may not work on a modern
   system without first installing the ``python-keystoneclient`` and
   ``python-neutronclient`` deb packages.
 
+> **NOTE**: The network type required for OVN is geneve - the default for the 
+  neutron-ext-net scripts is currently gre.
```

Comment:
I would say: "the default overlay network type".

https://opendev.org/openstack/neutron/src/commit/d444a8fbf073c96579987c8e934e7753a4150975/neutron/plugins/ml2/drivers/ovn/mech_driver/mech_driver.py#L310-L314